src/tools/dart-devtools.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Dart DevTools
+description: A suite of performance tools for Dart and Flutter.
+toc: false
+---
+
+Dart DevTools is a suite of performance tools for Dart and Flutter.
+DevTools supports source-level debugging of Dart command-line apps,
```

Comment:
There is nothing command-line specific about DevTools. You can use it for Flutter apps, command-line apps, server apps, and web apps. I suggest we just say 'Dart apps'.